sklearn/preprocessing/tests/test_target_encoder.py

```diff
@@ -135,6 +137,95 @@ def test_encoding(categories, unknown_value, global_random_seed, smooth, target_
     assert_allclose(X_test_transform, expected_X_test_transform)
 
 
+@pytest.mark.parametrize("smooth", [5.0, "auto"])
+def test_encoding_multiclass(global_random_seed, smooth):
```

Comment:
Could you please parametrize this test to use either integers or string values for the categorical features labels and the target class?